### PR TITLE
Add drop-down menu and back button.

### PR DESCRIPTION
**Pull request by duckonomy**
## Overview: ##
Add navigation to welcome page button and bring back the dropdown project selector in the control bar. 

## PR Status: ##

* [X] Ready.
* [ ] Work in Progress.
* [ ] Hold.

## Related Jira tickets: ##

* [DES-1953](https://jira.tacc.utexas.edu/browse/DES-1953)

## Summary of Changes: ##
Added the ui parts to the control-bar component.

## Testing Steps: ##
1. Confirm that routes work properly with the drop-down selector.
2. Confirm that the back button brings the user back to the welcome page.

## UI Photos:
Screenshot_2021-05-04 HazMapper

### CODE REVIEW

File: src/app/components/control-bar/control-bar.component.html [hunk 1]
```diff
@@ -6,6 +6,7 @@
         <i class="fas fa-spin fa-spinner"></i>
     </div>
     <div *ngIf="!loadingActiveProject && !loadingActiveProjectFailed" class="cell small-8 align-middle grid-x">
+      <button class="button tiny" (click)="routeToMain()"><i class="fas fa-arrow-left"></i></button>
```
Comment (nathanfranklin) [inline src/app/components/control-bar/control-bar.component.html:9]: should we hide if public view?

**Review by nathanfranklin — CHANGES_REQUESTED** [review]
LGTM but have one comment to address. 👍

File: src/app/components/control-bar/control-bar.component.html (hunk 1 above)
Comment (duckonomy) [reply src/app/components/control-bar/control-bar.component.html]: 👍🏼

**Review by nathanfranklin — APPROVED** [review]
👍 LGTM

**Pull request merged**